core/reflection.jvm/resources/META-INF/proguard/kotlinreflect.pro

```diff
@@ -0,0 +1,14 @@
+##---------------Begin: Kotlin-reflect ----------
```

Comment:
Are these Begin/End tags required?

Also, please consider renaming the file itself to `kotlin-reflect.pro`

Reply:
> Are these Begin/End tags required?

Not needed. They will be added automatically by R8 now for blaming purposes.

https://issuetracker.google.com/issues/134372926

Reply:
By R8 yes. Do we care about it in a proguard context? They're not strictly required but the blaming purpose is nice

Reply:
Removed in c6f922b